### DISCUSSION TITLE
Alternative to VPN_ENDPOINT_IP for DDNS hosts?

### DISCUSSION THREAD

**Discussion by sj14** [qdm12/gluetun#1093] [Q&A] [answered]
I'm running a Wireshark server on my home router and would like to connect using the DDNS address instead of the (changing) IP address. Is this possible?

**Comment by qdm12** [comment, marked as answer]
You would have to manually dns resolve your hostname to an ip address every time you want to connect for now (note ip usually change every few weeks only though). Subscribe to #788 to be alerted when this is done.
